File: ts/rick.ts
class Rick {
    message: string;

    constructor() {
        this.message = "";
    }

    never(): Rick {
        this.message += "Never ";
        return this;
    }

    gonna(): Rick {
        this.message += "gonna ";
        return this;
    }

    give(): Rick {
        this.message += "let ";
        return this;
    }

    you(): Rick {
        this.message += "you ";
        return this;
    }

    up() {
        console.log(this.message + "down!")
    }
}

let rick = new Rick();
rick.never().gonna().give().you().up()

File: python/rick.py
class Rick:
    def never(self) -> "Rick":
        print("never ", end="")
        return self

    def gonna(self) -> "Rick":
        print("gonna ", end="")
        return self

    def give(self) -> "Rick":
        print("let ", end="")
        return self

    def you(self) -> "Rick":
        print("you ", end="")
        return self

    def up(self) -> None:
        print("down!")


if __name__ == "__main__":
    rick = Rick()
    rick.never().gonna().give().you().up()
